Doc/library/pdb.rst

```diff
@@ -211,6 +211,9 @@ access further features, you have to do this yourself:
    The *readrc* argument defaults to true and controls whether Pdb will load
    .pdbrc files from the filesystem.
 
+   The *commands* argument, if given, would be a list of commands to execute
+   when the debugger starts. It has similar effects to the :file:`.pdbrc` file.
```

Comment:
```suggestion
   The *commands* argument, if given, is a list of commands to execute
   when the debugger starts. Those commands are executed after those
   optionally provided by the :file:`.pdbrc` file. 
```

Reply:
Do you think two "those" is a little bit confusing here? How about These commands are executed after any commands optionally provided by the :file:`.pdbrc` file?

Reply:
I agree the double "those" is confusing.

Reply:
Oups, you're right! I'm sorry I didn't reply to your question 3 weeks ago btw, but yes your suggestion is way better!